HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRPlayer.cs

```diff
@@ -727,6 +735,29 @@ private void GetPlayerConditions()
         _debuffCondition = (DebuffConditions)Process.Memory.Read<ulong>(conditionPtr + 0x38);
     }
 
+    [ScannableMethod]
+    private void GetPlayerActionArgs()
+    {
+        if (!InHuntingZone)
+        {
+            _actionFlag = ActionFlags.None;
+            return;
+        }
+
+        long actionFlagArray = Process.Memory.Read(
+            AddressMap.GetAbsolute("LOCAL_PLAYER_DATA_ADDRESS"),
+            AddressMap.Get<int[]>("PLAYER_ACTIONFLAG_OFFSETS")
+        );
+
+        if (actionFlagArray == 0)
```

Comment:
There's a helper extension function called `IsNullPointer()`, you can use that instead of checking if the value is 0, it's easier to read